components/compas-open/test/compas-open.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { expect, fixtureSync, html } from "@open-wc/testing";
+
+import { CompasOpenElement } from "../src/CompasOpen.js";
```

Comment:
`import type { CompasOpenElement }`

otherwise sonarcloud will fail because you're importing twice from `src/CompasOpen.js`